src/infection-model/index.tsx

```diff
@@ -14,6 +15,14 @@ export type CurveData = {
   staff: ndarray;
 };
 
+export type CurveFunctionInputs = Omit<
```

Comment:
So this whole operation means that `CurveFunctionInputs` has all fields in `EpidemicModelInputs` except for `rateOfSpreadFactor` but also has `rateOfSpreadCells` and `rateOfSpreadDorms` added. Right?

Reply:
that is correct